packages/types/src/registry.ts

```diff
@@ -21,18 +30,35 @@ import { SimulationService } from '@buf/penumbra-zone_penumbra.connectrpc_es/pen
  */
 
 export const typeRegistry = createRegistry(
```

Comment:
Add comment here instructing devs to add more services to this list as they are created in core